CenterForOpenScience/ember-osf-reviews
# [IN-63][Reviews] Use ember-osf queryHasMany

## Purpose

Use ember-osf version of `queryHasMany`

## Summary of Changes

- Update usage of `queryHasMany` to `model.queryHasMany`  instead of `queryHasMany` from reviews `store`.
- Delete store
- Update tests

## Side Effects / Testing Notes

These changes affect the preprint detail page and the moderation list page.
On the preprint detail page:
- Make sure it works as usual with a simple preprint.
- Create a preprint with more than 10 contributors, and see if the preprint detail page works
with no errors (please check the console as well.)

On the moderation list page:
- Simply make sure it works as usual, no errors.

## Ticket

[IN-63](https://openscience.atlassian.net/browse/IN-63)

# Reviewer Checklist
